Projects/Features/FindIDFeature/Sources/SchoolSelectButtonView.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+import DomainModule
+
+struct SchoolSelectButtonView: View {
+    @State var placeholderText = "학교를 선택해 주세요"
+    @Binding var schoolList: [SchoolEntity]
+    var onCommit: (SchoolEntity) -> Void
+
+    private var textColor: Color {
+        placeholderText == "학교를 선택해 주세요" ?
+            .GrayScale.gray5 :
+            .GrayScale.gray9
+    }
+
+    public init(
+        schoolList: Binding<[SchoolEntity]>,
+        onCommit: @escaping (SchoolEntity) -> Void = {_ in }
+    ) {
+        _schoolList = schoolList
+        self.onCommit = onCommit
+    }
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 10) {
+            HStack(alignment: .bottom) {
+                Text(placeholderText)
+                    .dmsFont(.text(.medium), color: textColor)
+
+                Spacer()
+
+                Menu {
+                    ForEach(schoolList) { schoolList in
+                        Button("\(schoolList.name)", action: {
+                            placeholderText = schoolList.name
+                            onCommit(schoolList)
+
```

Comment:
```suggestion
                    ForEach(schoolList) { school in
                        Button("\(school.name)", action: {
                            placeholderText = schoolList.name
                            onCommit(school)

```
혼동의 여지가 있어서 schoolList보다는 단일 단위로 school 어떠신지요?